apps/dotcms-ui/src/app/api/services/dot-router/dot-router.service.ts

```diff
@@ -25,8 +25,9 @@ export class DotRouterService {
                     previousUrl: this._routeHistory.url
                 };
             });
+        // force route reload whenever params change;
+        this.router.routeReuseStrategy.shouldReuseRoute = () => false;
```

Comment:
Are you 100% sure this is the way to implement this? I've seen some docs that it needs to create a custom class for this kind of implementation.

Reply:
right, moved to use, DotCustomReuseStrategyService